clang/tools/clang-linker-wrapper/LinkerWrapperOpts.td

```diff
@@ -243,3 +243,11 @@ Flags<[WrapperOnlyOption]>,  HelpText<"Embed LLVM IR for runtime kernel fusion">
 def sycl_dump_device_code_EQ : Joined<["--", "-"], "sycl-dump-device-code=">,
   Flags<[WrapperOnlyOption]>,
   HelpText<"Path to the folder where the tool dumps SPIR-V device code. Other formats aren't dumped.">;
+
+// Options to enable/disable device dynamic linking.
+def sycl_allow_device_image_dependencies : Flag<["--", "-"], "sycl-allow-device-image-dependencies">,
+  Flags<[WrapperOnlyOption, HelpHidden]>,
+  HelpText<"Allow dependencies between device code images">;
+def no_sycl_allow_device_image_dependencies : Flag<["--", "-"], "no-sycl-allow-device-image-dependencies">,
```

Comment:
I am not sure if we need the negative version when we are dealing with sub-tools

Reply:
Let's see what other reviewers think. I don't have a strong opinion either, just added both to be consistent with the driver.

Reply:
Just the enabling option for the tools should be enough, the option complexities of what a user can do via the command line warrants both the positive and negative variants there.